Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -55,6 +62,19 @@ struct SettingsView: View {
                     }
                 }
                 
+                Section(header: Text("Contributions")) {
+                    Text(infoViewModel.openSourceRemark)
```

Comment:
Can you remove this row?

Reply:
Section(header: Text("Contributions")) {
                    Text(infoViewModel.openSourceRemark)
                    
                    so should remove this section ??

Reply:
Yes please